### ISSUE TITLE
More complete example

### ISSUE THREAD

**Issue by sebpiq** [sebpiq/backbone.statemachine#5]
Write a more useful example for Backbone.Stateful views. e.g. a picture viewer.

**Comment by adyz** [comment]
Yeah, the examples folder is not working because of the missing dependencies. 
Including bower will help us install these easier and actually see how the script works.

**Comment by sebpiq** [comment]
Thanks for the info. I've now fixed the examples 6ba1852f077bad4f09994ad2c8e93d2f3a0a17af
